Jenkinsfile

```diff
@@ -160,5 +160,23 @@ pipeline {
         }
       }
     }
+    stage("Arm Build") {
+      agent { label 'arm4' }
+      options {
+        timeout(time: 8, unit: 'HOURS')
+        timestamps()
+      }
+      environment {
```

Comment:
Do we need to specify the jdk version since scala 2.12 only works on jdk 8?

Reply:
Yeah, I have installed the openjdk-8u252-b09 that the version is same as jdk_1.8_latest of apache infra. The reason is that I could not find appropriate version of jdk that can run on arm platform from the jenkins software of apache infra. [infra jdk matrix](https://cwiki.apache.org/confluence/display/INFRA/JDK+Installation+Matrix) . I have also installed the maven-3.6.3 that version is same as maven_3_latest of apache infra. I will take time to ensure the versions of jdk and maven are consistent with those of apache infra. In most cases, the arm4 node is just for kafka arm ci.